### ISSUE TITLE
Attach to a Fragment or Hide

### ISSUE THREAD

**Issue by aseolin** [oguzbilgener/CircularFloatingActionMenu#32]
Hello. I'm trying to attach the circular floating menu on a fragment, so the menu it's not visible on other fragments or activities. How to do that?

I can do a workaround by hiding the menu when the fragment/activitiy changes. How to hide the menu?

**Comment by deano2390** [comment]
Same here, attaching to the activity is not ideal because the menu might be specific to a fragment

**Comment by evasyuk** [comment]
guys, there is detach() mode available, but it doesn't work!

**Comment by mindrunner** [comment]
How exactly am I supposed to do this. What are the proper methods? I always end up having no button anymore after a screen orientation change. No attach/detach/show/hide/whatever lets the button appear again. Any hints how to do this?

**Comment by martipello** [comment]
same here a little example would be great really like this library except for it either appearing in every fragment or only when that fragment is called and not going away after frament is closed

**Comment by sidmanu** [comment]
Just move all the FAB related code to onCreateView() of your fragment. Then in onDetach(), call actionButton.detach();
